rts/lra/client/src/main/resources/META-INF/services/javax.ws.rs.client.ClientBuilder

```diff
@@ -1 +1 @@
-org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder
+org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl
```

Comment:
With this change we should be consistent in whole LRA impl. The same change should be done in https://github.com/jbosstm/narayana/blob/master/rts/lra/jaxrs/src/main/resources/META-INF/services/javax.ws.rs.client.ClientBuilder.
I think especially this particular change would be nice to be tracked as a jira.
FYI @xstefank

Reply:
If you agree on the change I will create the JIRA for this and move it to a specific PR.